### PR TITLE
finish task of PR

### PR DESCRIPTION
**Pull request by Boichenko1988**

### CODE REVIEW

**Review by EugeneMz13 — COMMENTED** [review]
Everything fine

**Review by MarkIlchenko — APPROVED** [review]
Good!

File: Program.cs [hunk 1]
```diff
@@ -20,3 +20,5 @@
 
 favorite.GiveCountry(favorCountry, capitolCountry);
```
Comment (EugeneMz13) [inline Program.cs:22]: All good

**Review by EugeneMz13 — APPROVED** [review]

**Review by sergiipopil — COMMENTED** [review]
All good

**Review by sergiipopil — APPROVED** [review]
Approve

**Pull request merged**